core/src/main/kotlin/api/VersionsRoute.kt

```diff
@@ -21,7 +21,6 @@ package org.eclipse.apoapsis.ortserver.core.api
 
 import io.github.smiley4.ktorswaggerui.dsl.routing.get
 
-import io.ktor.server.application.call
```

Comment:
Any idea why this was not catched by detekt?

Reply:
No, unfortunately not.